Sources/MongoSwift/MongoClient.swift

```diff
@@ -88,6 +124,12 @@ public class MongoClient {
     /// If command and/or server monitoring is enabled, indicates what event types notifications will be posted for.
     internal var monitoringEventTypes: [MongoEventType]?
 
+    /// Encoder whose options are inherited by databases derived from this client.
+    public let encoder: BSONEncoder
```

Comment:
what's the benefit to making these publicly accessible?

Reply:
Mainly so that a user can encode/decode stuff in the same manner they would when inserting into/reading from a given collection without actually reaching out to the db. Like if they read/write bson to/from another source (e.g. disk). Also, so they can inspect which options are set on a givent client/db/collection.

Reply:
> Like if they read/write bson to/from another source (e.g. disk)

should we rewrite bsondump in Swift? 😛